news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+The map feels considerably shorter than every other 10+ minute map, such as Nishizumi’s worldenddominator or Midnaait’s International OTOMAD’s 6. Even if you are not typically a fan of Demetori, I suggest you give this map a try, as it is a journey you will not forget.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/icaros.jpg)](http://osu.ppy.sh/beatmapsets/762660)
+
+[ARGENTINE DREAM](https://osu.ppy.sh/users/1981781) is no outsider to metal songs, and she gracefully demonstrates it with this new masterpiece on Emperor's "An Elegy Of Icaros".
+
+The unpredictability of the song does not stop her from masterfully composing a blend of long streams, intercalated 1/6 patterns, shameless usage of finishers and a very intelligent representation of specific musical concepts such as guitar tremolos and percussion flams, which are mapped as 1/8 doubles. Despite the song being repeated structurally, the sheer variation of meters (5/4-3/4 to simple 4/4), BPM and snaps add a vast amount of spice to an already spicy mix.
+
+In short, one of the best if not the best black metal maps to exist for now.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/161914.jpg)](http://osu.ppy.sh/beatmapsets/832971)
+
+"Unusual" would be my word choice to describe this set. Not only the song, which is a fairly uncommon eerie breakcore one, but also the song's title and artist, Sorry about my face's "161914". What do they mean? Will we ever know?
+
+Another unusual (but good!) trait of this set by [vrnl](https://osu.ppy.sh/users/4799788) is its ability to stay almost perfectly linear progression-wise while retaining the core aspects of the song, and this is, as you may imagine, not an easy task in a breakcore song. The highest difficulty manages to squeeze every single element the song has to offer, with 1/6 patterns reaching the astounding amount of 13 notes in a row!
+
+Overall, despite possibly needing some more spice in terms of slider velocity manipulation, the set accomplishes its objectives quite well: linear progression and challenging top difficulty. A well-deserved place in this season's Spotlights!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/border.jpg)](http://osu.ppy.sh/beatmapsets/732190)
+
+This great set by [\-Kazu-](https://osu.ppy.sh/users/920861) exemplifies what puts him head and shoulders above the competition. Extremely fun top diff that is very difficult but fair, this map harkens back to the very old days of IOSYS in it's difficulty/pattern homages.
+
+Kazu's often abstract mapping style really works for this kind of abrasive music, and generally we feel this map has great variation across all difficulties which allows you to appreciate all textures of the music. The lower difficulties maintain their identity without copying elements from the higher ones, and they all have distinct sections that players can appreciate.
```

Comment:
link to profile

Reply:
They are only hyperlinked once per comment, This is done in the previous paragraph.